### PR TITLE
Tooltip for disabled kebab is missing in Manage permissions page

### PR DESCRIPTION
**Pull request by YuliaKrimerman**
[RHOAIENG-13050](https://issues.redhat.com/browse/RHOAIENG-13050)

## Description

Added a "The default group always has access to model registry" Tooltip for disabled kebab in Manage permissions page when you hover over it
Screenshot 2024-10-16 at 11 14 12 AM

## How Has This Been Tested?

On the UI go to Model registry settings table in the Admin section. Click on Manage Permissions Verify whether the first group in "Groups" section in the Users tab has a disabled kebab menu and a tooltip like the screenshot is showing up when we hover over the disabled kebab. 

## Test Impact

Fixed the tests to pass with the Button now

## Request review criteria:

Self checklist (all need to be checked):
- [x] The developer has manually tested the changes and verified that the changes work
- [x] Testing instructions have been added in the PR body (for PRs involving changes that are not immediately obvious).
- [x] The developer has added tests or explained why testing cannot be added (unit or cypress tests for related changes)

If you have UI changes: 

- [x] Included any necessary screenshots or gifs if it was a UI change.
- [x] Included tags to the UX team if it was a UI/UX change.

After the PR is posted & before it merges:
- [ ] The developer has tested their solution on a cluster by using the image produced by the PR to `main`

### CODE REVIEW

File: frontend/src/concepts/roleBinding/RoleBindingPermissionsTableRow.tsx [hunk 1]
```diff
@@ -173,9 +178,18 @@ const RoleBindingPermissionsTableRow: React.FC<RoleBindingPermissionsTableRowPro
                 />
               </SplitItem>
             </Split>
+          ) : isDefaultGroup ? (
+            <Tooltip content="The default group always has access to model registry">
```
Comment (manaswinidas) [inline frontend/src/concepts/roleBinding/RoleBindingPermissionsTableRow.tsx:182]: full stop missing at the end according to [design](https://www.figma.com/proto/KTsYhANYQqB0NUrXW7wQH1/Model-Registry?page-id=6782%3A296930&node-id=8709-219955&viewport=1846%2C28%2C0.43&t=anh0eGvvgwFUHiMb-1&scaling=min-zoom&content-scaling=fixed&starting-point-node-id=8709%3A221017)

**Review by manaswinidas — CHANGES_REQUESTED** [review]
small nit otherwise lgtm

**Comment by manaswinidas** [comment]
@YuliaKrimerman can you add manual testing instructions in the "How has this been tested?" section. It will really be helpful for anyone outside of our scrum and/or QA revisits this. For example, Instructions like 1. Go to Model registry settings table in the Admin section 2. Go to Manage Permissions 3. Check whether the first group in "Groups" section in the Users tab has a disabled kebab menu and a tooltip like the screenshot.

File: frontend/src/concepts/roleBinding/RoleBindingPermissionsTableRow.tsx (hunk 1 above)
Comment (YuliaKrimerman) [reply frontend/src/concepts/roleBinding/RoleBindingPermissionsTableRow.tsx]: Good catch! Added.

**Review by manaswinidas — APPROVED** [review]
/lgtm 

Tested whether the tooltip is present.

**Comment by yih-wang** [comment]
LGTM, thanks @YuliaKrimerman !

**Review by mturley — APPROVED** [review]
LGTM. Shame that we can't just pass a `disabledTooltip` into ActionsColumn, but that's a PF problem for another time and maybe not worth addressing.

**Pull request merged**